CRM/Core/Resources/Common.php

```diff
@@ -204,13 +205,21 @@ protected static function coreResourceList($region) {
     ];
 
     // Dynamic localization script
-    $items[] = Civi::resources()->addCacheCode(
-      CRM_Utils_System::url('civicrm/ajax/l10n-js/' . CRM_Core_I18n::getLocale(),
-        ['cid' => CRM_Core_Session::getLoggedInContactID()], FALSE, NULL, FALSE)
-    );
+    $items[] = Civi::service('asset_builder')->getUrl('crm-l10n.js', [
+      'cid' => $contactID,
+      'includeEmailInName' => (bool) $settings->get('includeEmailInName'),
+      'ajaxPopupsEnabled' => (bool) $settings->get('ajaxPopupsEnabled'),
+      'allowAlertAutodismissal' => (bool) $settings->get('allow_alert_autodismissal'),
+      'resourceCacheCode' => Civi::resources()->getCacheCode(),
+      'locale' => CRM_Core_I18n::getLocale(),
+      'lcMessages' => $settings->get('lcMessages'),
+      'dateInputFormat' => $settings->get('dateInputFormat'),
+      'timeInputFormat' => $settings->get('timeInputFormat'),
+      'moneyFormat' => CRM_Utils_Money::format(1234.56),
```

Comment:
@colemanw I notice that the original code json_encoded this value but your not anymore?